webui/react/src/components/Json.tsx

```diff
@@ -27,17 +27,19 @@ const row = (
   }
   return <li className={css.item} key={label}>
     <span className={css.label}>
-      {typeof label === 'string' && transformLabel ? transformLabel(label) : label}
+      {typeof label === 'string' && translateLabel ? translateLabel(label) : label}
       :</span>
-    <span className={css.value}>{textValue}</span>
+    <span className={css.value}>
+      {textValue}
+    </span>
   </li>;
 };
 
-const Json: React.FC<Props> = ({ json, transformLabel }: Props) => {
+const Json: React.FC<Props> = ({ json, translateLabel }: Props) => {
 
   return (
     <ul className={css.base}>
-      {Object.entries(json).map(([ label, value ]) => row(label, value, transformLabel))}
+      {Object.entries(json).map(([ label, value ]) => row(label, value, translateLabel))}
```

Comment:
**suggestion:** `row` should probably just be a react function component `JsonRow` inside this same file.